packages/block-editor/src/store/selectors.js

```diff
@@ -2767,6 +2767,17 @@ export function __unstableGetTemporarilyEditingAsBlocks( state ) {
 	return state.temporarilyEditingAsBlocks;
 }
 
+/**
+ * DO-NOT-USE in production.
+ * This selector is created for internal/experimental only usage and may be
+ * removed anytime without any warning, causing breakage on any plugin or theme invoking it.
+ *
+ * @param {Object} state Global application state.
+ */
+export function __unstableGetTemporarilyEditingFocusModeToRevert( state ) {
+	return state.temporarilyEditingFocusModeRevert;
+}
```

Comment:
Why was this not added as a private selector?

Reply:
@jorgefilipecosta: ping, in case you missed this :)

Reply:
Sorry, I missed this comment. I followed what was happening for other content-locking selectors, but yes I guess all of them could be private. I will prepare a PR.